### PR TITLE
Migrating FeesAction getFeeCategoryHeadWise, getDndReport, deleteOtherFeesCategory

### PR DESCRIPTION
**Pull request by Tauqueer031**

### CODE REVIEW

File: src/main/java/org/ideoholic/curium/model/feescategory/service/FeesService.java [hunk 1]
```diff
@@ -711,24 +712,24 @@ private int getLedgerAccountId(String itemAccount) {
...
+	        if(branchid!=null){
+	        	String[] yearofAdmission = yearofAdmissionStr.split("/");
+	        	String[] currentAcademicYear = currentAcademicYearStr.split("/");
 	        	String searchYear = null;
 	        	int yoa = Integer.parseInt(yearofAdmission[0]);
 	        	int ca = Integer.parseInt(currentAcademicYear[0]);
 	        	
 	        	if(yoa == ca || yoa < ca) {
-	        		searchYear = httpSession.getAttribute("currentAcademicYear").toString();
+	        		searchYear = currentAcademicYearStr.toString();
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/feescategory/service/FeesService.java]: `searchYear = currentAcademicYearStr;`

File: src/main/java/org/ideoholic/curium/model/feescategory/action/FeesActionAdapter.java [hunk 2]
```diff
@@ -196,5 +197,29 @@ public String applyotherConcession() {
...
+	}
+
+	public String deleteOtherFeesCategory() {
+		FeesService feesService = new FeesService(request, response);
+		ConcessionDto concessionDto = new ConcessionDto();
+		concessionDto.setSfsid(request.getParameterValues("sfsid"));
+		concessionDto.setId(request.getParameter("id"));
+		StudentIdDto studentIdDto = feesService.deleteOtherFeesCategory(concessionDto);
+		String studentId = studentIdDto.getStudentId();
+		return studentId;
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/feescategory/action/FeesActionAdapter.java]: `return studentIdDto.getStudentId()`

**Review by reehanlalkhan — CHANGES_REQUESTED** [review]

File: src/main/java/org/ideoholic/curium/model/feescategory/service/FeesService.java (hunk 1 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/feescategory/service/FeesService.java]: done

File: src/main/java/org/ideoholic/curium/model/feescategory/action/FeesActionAdapter.java (hunk 2 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/feescategory/action/FeesActionAdapter.java]: done

**Review by reehanlalkhan — APPROVED** [review]

**Pull request merged**